### ISSUE TITLE
AdmissionReview spec undocumented

### ISSUE THREAD

**Issue by RichardoC** [kubernetes/website#39174]

**Comment by sftim** [comment]
Does https://kubernetes.io/docs/reference/config-api/apiserver-admission.v1/#admission-k8s-io-v1-AdmissionReview help? Maybe it'd be useful if some other pages linked there.

**Comment by RichardoC** [comment]
> Does https://kubernetes.io/docs/reference/config-api/apiserver-admission.v1/#admission-k8s-io-v1-AdmissionReview help? Maybe it'd be useful if some other pages linked there.

Yes, that's exactly what I was hoping to find from the API one pager. Thank you!

**Comment by sftim** [comment]
We've got two API reference generators right now, and we'd love to combine the best parts of both. It's not a massive undertaking, but it's not small either - and we've got more urgent priorities.

**Comment by sftim** [comment]
@RichardoC are you happy to see this closed? We could add the extra links; any thing more involved is covered by bigger / long term issues, I think.

**Comment by RichardoC** [comment]
I'm happy for this to be closed, if links to these request/response formats are added to Flux/2 where is shows example request/responses. I can raise the PR if you'd prefer?

Want me to comment on the issue about the API generation consolidation?

**Comment by tengqm** [comment]
To clarify, the APIs under [https://kubernetes.io/docs/reference/config-api/](https://kubernetes.io/docs/reference/config-api/) are supplemental to the public APIs. They are currently called "config APIs" for lacking of a better term. These APIs are not included in the public APIs, i.e. the swagger or OpenAPI v3 spec generated from the upstream k/k repository.
Yes, we do have two API reference generators that consume the "public" APIs but produce different formats. They serve for people with different tastes. And, this is orthogonal to the `AdmissionReview` API issue here.

/close
